axelor-base/src/main/resources/views/Partner.xml

```diff
@@ -541,7 +541,8 @@
 					if-module="axelor-sale"
 					if="__config__.app.isApp('sale') &amp;&amp; __config__.app.getApp('sale').getEnableCustomerCatalogMgt()" />
 				<panel name="accountblockedPanel">
-					<field name="hasBlockedAccount"/>
+					<field name="hasBlockedAccount" onChange="action-attrs-update-manually-blocked-account"/>
+					<field name="hasManuallyBlockedAccount" onChange="action-attrs-update-blocked-account"/>
```

Comment:
I think it would be better to simply hide hasManuallyBlockedAccount and remove action-attrs-update-blocked-account